sky/provision/gcp/instance_utils.py

```diff
@@ -1406,10 +1403,23 @@ def create_tpu_node(project_id: str, zone: str, tpu_node_config: Dict[str, str],
             _log_errors(provisioner_err.errors, e, zone)
             raise provisioner_err from e
 
+        provisioner_err.errors = [{
+            'code': 'UNKNOWN',
```

Comment:
Minor: is it possible `e/stderr` actually have a different code? Can simply leave a TODO since it's pretty minor.

Reply:
Added the TODO. Thanks!